### DISCUSSION TITLE
Inaccuracies in taiko imported scores

### DISCUSSION THREAD

**Discussion by fishy15** [ppy/osu#25882] [Ideas] [answered]
It seems like the imported scores for taiko may have incorrect ranks? Specifically, it seems like the rank given is based on the stable scoring system rather than the cutoffs in lazer. This leads to some inconsistent UI as in the 2nd screenshot, where the accuracy wheel shows an A rank, but the actual rank shown is a B.

Additionally, the accuracies seem to use the stable weights instead of lazer weights, which gives an accuracy of ~93% vs ~91% as seen below in the 3rd screenshot. The score also seems to be off by about 50%, but if I understand correctly, this is due to the 0.5x multiplier on classic mode so it should not be an issue. It might be useful to indicate this to users somehow so they are not confused about why there is such a large difference in scores though. 

<img width="1680" alt="song select with inaccurate rank given" src="https://github.com/ppy/osu/assets/38575855/8363e9f6-acc4-4700-ade7-caf153d1f710">

directly viewing results screen:
<img width="1680" alt="results screen of top score with inaccurate rank given" src="https://github.com/ppy/osu/assets/38575855/7c239257-5cc4-4cd6-8683-810508e1e623">

viewing results screen after replay:
<img width="1680" alt="results screen of top score after replay" src="https://github.com/ppy/osu/assets/38575855/e2749ff3-f7ae-43e3-a7ad-4eda00299882">

[replay of top play, change extension to .osr](https://github.com/ppy/osu/files/13710086/fishy15.playing.Wakeshima.Kanon.-.Tsukinami.Sebola.Nostalgia.2023-11-18_10-44.txt)

**Comment by peppy** [comment, marked as answer]
This is going to be the same thing as https://github.com/ppy/osu/issues/9087.
